lib/llm/src/http/service/health.rs

```diff
@@ -37,17 +37,44 @@ pub fn health_check_router(
     state: Arc<service_v2::State>,
     path: Option<String>,
 ) -> (Vec<RouteDoc>, Router) {
-    let path = path.unwrap_or_else(|| "/health".to_string());
+    let health_path = path.unwrap_or_else(|| "/health".to_string());
 
-    let docs: Vec<RouteDoc> = vec![RouteDoc::new(Method::GET, &path)];
+    let docs: Vec<RouteDoc> = vec![RouteDoc::new(Method::GET, &health_path)];
 
     let router = Router::new()
-        .route(&path, get(health_handler))
+        .route(&health_path, get(health_handler))
         .with_state(state);
 
     (docs, router)
 }
 
+pub fn live_check_router(
+    state: Arc<service_v2::State>,
+    path: Option<String>,
+) -> (Vec<RouteDoc>, Router) {
+    let live_path = path.unwrap_or_else(|| "/live".to_string());
+
+    let docs: Vec<RouteDoc> = vec![RouteDoc::new(Method::GET, &live_path)];
+
+    let router = Router::new()
+        .route(&live_path, get(live_handler))
+        .with_state(state);
+
+    (docs, router)
+}
+
+async fn live_handler(
+    axum::extract::State(_state): axum::extract::State<Arc<service_v2::State>>,
+) -> impl IntoResponse {
+    (
+        StatusCode::OK,
+        Json(json!({
+            "status": "live",
+            "message": "Service is live"
+        })),
+    )
+}
```

Comment:
Can we only add this when it will be accurate?

Reply:
What does the HTTP server do if something in it's handler panics?

Should you check if there are any model entries, and only return "live" if there are? I'm not sure what Kubernetes is expecting from this endpoint.